api/r/CellCensus/R/open.R

```diff
@@ -5,26 +5,30 @@ DEFAULT_TILEDB_CONFIGURATION <- c(
 
 #' Open the Cell Census
 #'
-#' @param census_version The version of the Census, e.g., "latest"
+#' @param census_version The version of the Census, e.g., "latest".
 #' @param uri The URI containing the Census SOMA objects. If specified, takes
 #'            precedence over `census_version`.
+#' @param s3_region Optional AWS region accompanying uri.
 #'
 #' @return Top-level `tiledbsoma::SOMACollection` object
 #' @importFrom tiledbsoma SOMACollection
 #' @importFrom tiledb tiledb_ctx
 #' @export
 #'
 #' @examples
-open_soma <- function(census_version = "latest", uri = "") {
+open_soma <- function(census_version = "latest", uri = "", s3_region = "") {
```

Comment:
is `s3_region` a temporary placeholder for passing a full `context`?   At least in the Python API, the equivalent function accepts a context, which contains any necessary TileDB configuration.

If a placeholder, please disregard this comment.  If not, should it instead be a `context`?

Reply:
`SOMATileDBContext` was *just* added to the R API over the weekend, so I'll do a follow-up to receive and pass it around!